src/components/Popover/PopoverContent.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { ReactNode } from 'react';
+import ReachUIPopover from '@reach/popover';
+import { useForkedRef } from '@reach/utils';
+import { animated, useTransition } from 'react-spring';
+import Card from '../Card';
+import { usePopoverContext } from './Popover';
+import usePopoverContentAlignment from './usePopoverContentAlignment';
+
+const defaultOffset = [0, 12] as [number, number];
+const AnimatedPopover = animated(ReachUIPopover);
+
+export interface PopoverContentProps {
+  /**
+   * The position to expand the popover
+   *
+   * @default left
+   */
+  alignment?: 'bottom-left' | 'bottom-center' | 'bottom-right' | 'left-center' | 'right-center';
+
+  /**
+   * The spacing between the trigger and the popover in px, for the horizontal & vertical axis
+   *
+   * @default [0, 12]
+   */
+  offset?: [number, number];
+
+  /**
+   * @ignore
+   */
+  children: ReactNode;
+}
+
+const PopoverContent = React.forwardRef<HTMLDivElement, PopoverContentProps>(
+  function PopoverContent(
+    { alignment = 'bottom-left', offset = defaultOffset, children, ...rest },
+    forwardedRef
+  ) {
+    const popoverAlignment = usePopoverContentAlignment({ alignment, offset });
+    const { popoverId, isOpen, triggerRef, popoverRef, close } = usePopoverContext();
+
+    // Close on clicks outside. We only want to close if a click occurred outside the popover
+    // and its trigger. We also add `capture` events to avoid some race conditions on window-attached
+    // events
+    React.useEffect(() => {
```

Comment:
I guess we can move this part to a hook in order to control and reuse the behavior.

Reply:
Implemented